### PR TITLE
LPS-145899 | LPS-146134 | LPS-146239

### PR DESCRIPTION
**Pull request by carolmariaabb**

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java [hunk 1]
```diff
@@ -482,6 +482,18 @@ private void _validate(
...
+
+		if (objectDefinition.isSystem() &&
+			!Objects.equals(
+				type, ObjectRelationshipConstants.TYPE_ONE_TO_MANY)) {
+
+			throw new ObjectRelationshipTypeException(
+				"Invalid type for system object definition " +
+					objectDefinitionId1);
+		}
+
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java:496]: I created this ticket [LPS-146355](https://issues.liferay.com/browse/LPS-146355) related to the comment above. Feel free to close if it's not a bug.

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (gabrielwas) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java:496]: Tottaly agree, @carolmariaabb! Thanks!

**Comment by gabrielwas** [comment]
ci:forward

**Pull request closed without merging**